### PR TITLE
Add Job Run Task List

### PR DESCRIPTION
**Pull request by rupurt**
[Finishes #161557980]

### CODE REVIEW

**Comment by NavyAdmiral** [comment]
In case the params are empty due to some reason, wouldn't it be better to display some message for a better experience? Attaching a picture that confuses me
image

File: gui/src/components/JobRuns/StatusItem.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  return <Typography>{summary}</Typography>
+}
+
+const StatusItem = ({status, summary, borderTop, children, classes}) => (
+  <div className={classNames(classes.item, {[classes.borderTop]: borderTop})}>
+    <div className={classes.status}>
+      <StatusIcon>{status}</StatusIcon>
+    </div>
+    <div className={classes.details}>
+      <Grid container>
```
Comment (NavyAdmiral) [inline gui/src/components/JobRuns/StatusItem.js]: Are the grids in lines 76,77,79,80 necessary?

File: gui/src/components/JobRuns/StatusItem.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    details: {
+      padding: theme.spacing.unit * 2
+    },
+    expansionPanel: {
+      boxShadow: 'none'
+    }
+  }
+}
+
+const render = (summary, borderTop, children, classes) => {
```
Comment (NavyAdmiral) [inline gui/src/components/JobRuns/StatusItem.js]: Isn't borderTop unused?

File: gui/src/components/JobRuns/StatusItem.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+const styles = theme => {
+  return {
+    borderTop: {
+      borderTop: 'solid 1px',
+      borderTopColor: theme.palette.divider
+    },
+    item: {
+      position: 'relative',
+      paddingLeft: 50
```
Comment (NavyAdmiral) [inline gui/src/components/JobRuns/StatusItem.js]: Wasn't it suggested to avoid this and use Material UI's spacings ?

**Comment by rupurt** [comment]
@NavyAdmiral fair call on the empty params. I'll raise it with our designer.

File: gui/src/components/JobRuns/StatusItem.js (hunk 3 above)
Comment (rupurt) [reply gui/src/components/JobRuns/StatusItem.js]: 👍 This value is fixed and not related to the theme spacing. I'll add it to the theme though as a custom value.

File: gui/src/components/JobRuns/StatusItem.js (hunk 1 above)
Comment (rupurt) [reply gui/src/components/JobRuns/StatusItem.js]: 👍 not required

**Pull request merged**